book.rb

```diff
@@ -0,0 +1,13 @@
+class Book
+  attr_accessor :title, :author, :rentals
+
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = [] # Create the has-many side (a book has many rentals).
+  end
+
+  def add_rental(rental)
+    @rentals << rental # The << operator is used to push an element into an array.
+  end
```

Comment:
- [ ] The book itself should be sent as a parameter to create the new `rental` achieving the 'has-many' association. 

### Sample code:
```
 def add_rental(person, date) 
    Rental.new(date, self, person)
  end
```